esp/src/eclwatch/InfoGridWidget.js

```diff
@@ -427,6 +437,10 @@ define([
                         return -1;
                     } else if (r.Severity === "Warning") {
                         return 1;
+                    } else if (l.Severity === "Alert") {
```

Comment:
Given your coloring alerts the same as errors, they should probably get sorted above warnings, but after errors?